app/main.py

```diff
@@ -1,34 +1,64 @@
-class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+from typing import List, Tuple, Dict
+
 
+class Deck:
+    def __init__(self, row: int, column: int,
+                 is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
```

Comment:
According to the checklist item 4, you have created an attribute 'is_alive' in the class 'Deck' but it seems like you are not using it anywhere in your code. If it's not needed, consider removing it to improve the code quality.